docs/creating.md

```diff
@@ -95,27 +95,40 @@ suitable to [autoupdate your project safely][the-copier-answersyml-file]:
+| `settings`         | [`copier.settings.Settings`][]                                | General user settings.<br>See the [`settings`][] page for related information.                                                                                                                                                                             |
+| `skip_answered`    | `bool`                                                        | When `True`, skip questions that have already been answered.<br>See the [`skip_answered`][] setting for related information.                                                                                                                               |
+| `skip_if_exists`   | `Sequence[str]`                                               | Specified additional [file skip patterns][patterns-syntax].<br>See the [`skip_if_exists`][] setting for related information.                                                                                                                               |
+| `skip_tasks`       | `bool`                                                        | When `True`, skip [template tasks execution][tasks].<br>See the [`skip_tasks`][] setting for related information.                                                                                                                                          |
+| `src_path`         | `Path`                                                        | The absolute path to the (cloned/downloaded) template on disk.                                                                                                                                                                                             |
+| `unsafe`           | `bool`                                                        | When `True`, allow usage of unsafe templates.<br>See the [`unsafe`][] setting for related information.                                                                                                                                                     |
+| `use_prereleases`  | `bool`                                                        | When `True`, `vcs_ref`/`vcs_ref_hash` may refer to a prerelease version of the template.<br>See the [`use_prereleases`][] setting for related information.                                                                                                 |
+| `user_defaults`    | `dict[str, Any]`                                              | Specified user defaults that may override a template's defaults during question prompts.                                                                                                                                                                   |
```

Comment:
Should we point to https://copier--2127.org.readthedocs.build/en/2127/settings/#user-defaults? What would that be, `See the [User defaults][settings.user-defaults] setting for related information.`